Dockerfile

```diff
@@ -0,0 +1,20 @@
+# ci-oracle-client-runner - Dockerfile for Oracle Instant Client
+
+FROM oraclelinux:7-slim
+
+ARG release=19
+ARG update=3
+
+# Install Oracle client packages from Oracle Linux yum server:
+RUN  yum -y install oracle-release-el7 && yum-config-manager --enable ol7_oracle_instantclient && \
+     yum -y install oracle-instantclient${release}.${update}-basic oracle-instantclient${release}.${update}-devel oracle-instantclient${release}.${update}-sqlplus oracle-instantclient${release}.${update}-tools && \
+     rm -rf /var/cache/yum
+
+# Path update required for tools package:
+ENV PATH=$PATH:/usr/lib/oracle/${release}.${update}/client64/bin
+
+# Additional packages used in scripts:
+RUN  yum install -y hostname-3.13-3.el7.x86_64 && \
```

Comment:
The package command here has the potential to fail in future if the RPM release version changes (i.e. if the RPM is patched). Instead, specify the version without the release number and architecture, for example `hostname-3.13`, to ensure the software version is maintained but the latest RPM edition is preferred (often older RPMs aren't maintained for the same software version).

Reply:
Done.